gsoc_contributors_guide.md

```diff
@@ -5,10 +5,104 @@ The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](http
+how netcdf defines common terms such as groups, dimensions, variables, and attributes.  
+
+* [Zarr](https://zarr.readthedocs.io/en/stable/) - a format for the storage of chunked, compressed, N-dimensional 
+arrays. Zarr has similar characteristics to netcdf, but has the added benefit of being a cloud-native data format. For 
+this reason, Zarr is ideal for storing large data sets in the cloud. Zarr provides a great overview of its [storage specifications](https://zarr.readthedocs.io/en/stable/spec/v2.html#hierarchies)
+that may be useful to read.    
+
+### Data Structures
+
+Netcdf and Zarr formats can be easily read with the [xarray](https://xarray.pydata.org/en/stable/index.html) library in 
+Python. Additionally, xarray enables efficient computation of our data, which is labelled and multi-dimensional. 
+A fantastic [xarray tutorial](https://xarray-contrib.github.io/xarray-tutorial/) has been put 
+together that describes the fundamentals of xarray. Be sure to become familiar with both DataArrays and Datasets as they
+are heavily used.     
+
+### Ocean Sonar Data
+
+For this project, you will be initially working with the output of the [compute_Sv](https://echopype.readthedocs.io/en/stable/api/echopype.calibrate.compute_Sv.html#echopype.calibrate.compute_Sv) 
+function. This is a function in echopype that computes the volume backscattering strength (Sv) of the provided EchoData
+object. This function returns an xarray Dataset that has several variables that are necessary for the visualization 
+of ocean sonar data. The Dataset has the dimensions and coordinates: 
+
+* frequency - transducer frequency used for the experiment, with units Hz
+
+* ping_time - timestamp of each ping
+
+* range_bin - sample index along a range
+
+The data variables of the Dataset are as follows, where items in parenthesis are the dimensions of the data variables: 
+
+* Sv (frequency, ping_time, range_bin) - volume backscattering strength measured from the echo
+
+* range (frequency, ping_time, range_bin) - the measured range of an echo in meters 
+
+* temperature - the temperature measurement of the water collected by the echosounder, with unit degree Celsius
+
+* salinity - the salinity measurement of the water collected by the echosounder, with unit part per thousand (PSU)
+
+* pressure - the pressure measurement of the water collected by the echosounder, with unit dbars
+
+* sound_speed (frequency, ping_time) - sound speed (in units m/s) for the provided temperature, salinity, and pressure 
+
+* sound_absorption (frequency, ping_time) - sea water absorption (in units dB/m) for each frequency and ping time, this 
+value is based on the temperature, salinity, pressure, and sound_speed
+
+* sa_correction (frequency) - the sa correction for each frequency
+
+* gain_correction - (frequency) - the gain correction for each frequency
+
+* equivalent_beam_angle (frequency) - the beam angle for each frequency
+
```

Comment:
```suggestion
Note that the key data variables you will be working with are `Sv` and `range`. All other variables are included in this dataset so that the parameters used in the calibration (from raw to Sv) are recorded.
```